docs/install/knative-with-any-k8s.md

```diff
@@ -572,7 +572,21 @@ To learn more about the Cloud Audit Logs source, try [our sample](../eventing/sa
 
 {{< /tab >}}
 
-<!-- TODO: couchdb source -->
+{{% tab name="CouchDB Source" %}}
```

Comment:
Apache CouchDB 

is trademark compliant usage

Reply:
indeed!